app/controllers/api/v1/members_controller.rb

```diff
@@ -3,6 +3,27 @@
 module Api
   module V1
     class MembersController < ApplicationController
+      before_action :set_member, only: %i[update]
+
+      def update
+        if @member.update(member_params)
+          render json: MemberSerializer.new(@member).serializable_hash.to_json, status: :ok
+        else
+          render json: @member.errors, status: :unprocessable_entity
+        end
+      end
+
+      private
+
+      def member_params
+        params.permit(:name, :description, :facebook_url, :instagram_url, :linkedin_url)
+      end
+
+      def set_member
+        @member = Member.find(params[:id])
```

Comment:
Taking into account that you need to check that the member exists, it would be better to write:
```suggestion
        @member = Member.kept.find(params[:id])
```